### PR TITLE
FDPIN-2661 ORC-264 backport, orc and hive version bumped

### PR DESCRIPTION
**Pull request by amrk7s**

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/io/orc/FDPSchemaEvolOrcInputFormat.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ *     long currentTransaction;
+ *     ROW row;
+ *   }
+ * </pre>
+ * Each AcidEvent object corresponds to an update event. The
+ * originalTransaction, bucket, and rowId are the unique identifier for the row.
+ * The operation and currentTransaction are the operation and the transaction
+ * that added this event. Insert and update events include the entire row, while
+ * delete events have null for row.
+ */
```
Comment (piyush-mukati-zz) [inline ql/src/java/org/apache/hadoop/hive/ql/io/orc/FDPSchemaEvolOrcInputFormat.java]: we should comment here why we added this class.

Comment (amrk7s) [reply ql/src/java/org/apache/hadoop/hive/ql/io/orc/FDPSchemaEvolOrcInputFormat.java]: This file addition goes into a separate commit 7b82029313e561ac7ae024832be8f5d03c913d11 with a description. But makes sense to add comment anyways.

**Comment by amrk7s** [comment]
Added comments to FDPSchemaEvalOrcInputFormat.java

**Review by piyush-mukati-zz — APPROVED** [review]
LGTM

**Pull request merged**